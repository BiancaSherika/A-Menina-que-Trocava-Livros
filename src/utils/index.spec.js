import { onNavigate } from './history.js';

describe('onNavigate', () => {
  it('should be a function', () => {
    expect(typeof onNavigate).toBe('function');
  });
  it('should load the page', () => {
    expect(onNavigate()).toMatchSnapshot();
  });
});
