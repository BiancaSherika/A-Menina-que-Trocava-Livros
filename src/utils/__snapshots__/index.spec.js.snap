// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`onNavigate should load the page 1`] = `undefined`;
